removeDuplicates.c

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
```

Comment:
Not quite what we were looking for - you returned an array with the 0s replacing the duplicates, but how about an implementation that would return a new array with the adjusted size? Any ideas how to go about that?